addons/website_sale/views/templates.xml

```diff
@@ -853,6 +853,155 @@
         </t>
     </template>
 
+    <template id="o_wsale_offcanvas_select_attribute" name="Select type attribute in Filter">
```

Comment:
Try to find a more suitable name (This applies to other templates you added too)
```suggestion
    <template id="website_sale.filter_select_attributes" name="Select attributes in Filter">
```